extensions/azurecore/src/account-provider/azureAccountProvider2.ts

```diff
@@ -189,6 +189,8 @@ export class AzureAccountProvider implements azdata.AccountProvider {
 		nonce: string,
 		authUrl: string) {
 
+		const mediaPath = path.join(__dirname, '..', '..', 'media');
```

Comment:
Any reason you're not using vscode.ExtensionContext.extensionPath instead? Would be a safer way of knowing you're at the root level. 

Because right now isn't this still going to break for webpacked builds? __dirname for that will be `resources\app\extensions\azurecore\dist` so it'll resolve to `resources\app\extensions\media\` which isn't right.

Reply:
Because I didn't know that was a method :D